manifests/bucketeer/charts/experiment-calculator/values.yaml

```diff
@@ -80,11 +80,11 @@ health:
   # The duration is up to resources.
   # cf. CPU: 500m, MEM: 3Gi -> 280sec
   livenessProbe:
-    initialDelaySeconds: 300
```

Comment:
The service takes around 4-5 minutes to get ready. Is there a reason to change it?

Reply:
Remember that I have create a PR for httpstan that precompile the stan model: https://github.com/bucketeer-io/bucketeer/pull/454?

For now, the service will be ready in around 10s. So if we still set this to 300s, we need to wait 300s for the pod to be ready.

Reply:
Yes, I remembered. It improved a little, but it took more than 3 minutes in production to get ready. So, the health check restarts the container before it gets ready.

I will check it on the dev environment to ensure that it gets ready in 10 seconds.